### ISSUE TITLE
Vyper not handling braces in mathematical expressions and calls in one expression properly

### ISSUE THREAD

**Issue by michwill** [vyperlang/vyper#1615]
### Version Information

* vyper Version (output of `vyper --version`): 0.1.0b12
* OS: linux
* Python Version (output of `python --version`): 3.6.8
* Environment (output of `pip freeze`): meh

### What's your issue about?

```
from vyper import compile_code                                                                                                                         

compile_code(""" 
@private 
@constant 
def f(x: uint256) -> uint256: 
    return x + 1 

@public     
@constant 
def g() -> uint256: 
    return self.f(1) * (self.f(20) - self.f(10)) 
""")
```
Outputs:
```
    433 
    434         if label_name in existing_labels:
--> 435             raise Exception('Label with name %s already exists!', label_name)
    436         else:
    437             existing_labels.add(label_name)

Exception: ('Label with name %s already exists!', 'save_locals_start_10_37')
```

**Comment by charles-cooper** [comment]
That's a bit strange, I don't have any issue compiling this as a standalone file from the command line

**Comment by charles-cooper** [comment]
Never mind. No trouble generating IR, the issue happens when translating to asm

**Comment by charles-cooper** [comment]
This seems fixed as of master(cbc9e65decbb)

```
$ vyper /dev/stdin <<EOF
> @internal
> @view
> def f(x: uint256) -> uint256: 
>     return x + 1 
> 
> @internal
> @view
> def g() -> uint256: 
>     return self.f(1) * (self.f(20) - self.f(10)) 
> EOF
0x61014f56600436101561000d57610018565b600035601c52600051505b60006000fd5b61016052610140526101405160018181830110610146578082019050905060005260005161016051565b6101405261014051600161016052610160516006580161001e565b6101c052610140526101c0516101405161016051610180516101a0516101c05160146101e0526101e0516006580161001e565b610240526101c0526101a052610180526101605261014052610240516101405161016051610180516101a0516101c0516101e051610200516102205161024051600a61026052610260516006580161001e565b6102c0526102405261022052610200526101e0526101c0526101a0526101805261016052610140526102c0518082106101465780820390509050808202821582848304141715610146578090509050905060005260005161014051565b600080fd5b61000461014f0361000460003961000461014f036000f3
```
